src/gatsby/src/components/search/filter.js

```diff
@@ -58,10 +58,21 @@ const Filter = ({ filterItems, selectedFilterItems, handleFilterCheck, filterTyp
               checked={checked}
               onChange={event => {
                 handleFilterCheck(item, event)
+                if (typeof window.snowplow === 'function') {
+                  window.snowplow(
+                    "trackStructEvent",
+                    "Checkbox",
+                    event.target.checked ? "Check" : "Uncheck",
+                    `${item.filterType} - ${item.label}`,
+                    "filter-checkbox",
```

Comment:
I would change this to `filter-type` and use it to pass `item.filterType` as a value. This allows grouping by the label alone without the group.